src/components/Search.js

```diff
@@ -1312,7 +1313,22 @@ class Search extends Component {
                     </Affix>
                 ) : null}
 
+                {Object.keys(logoSettings).length ? (
+                    <div>
+                        <img
+                            src={logoSettings.logoUrl}
+                            alt="logo-url"
+                            style={{
+                                width: `${logoSettings.logoWidth}px`,
+                                height: `${logoSettings.logoWidth}px`,
```

Comment:
```suggestion
```

Reply:
Why should the height be the same as the width? Typically, navbar height is fixed.